sepolicy/vendor/property_contexts

```diff
@@ -2,3 +2,4 @@
 #hw.cabl.                    u:object_r:hw_cabl_prop:s0
 #ro.opengles.                u:object_r:opengles_prop:s0
 persist.vendor.vis.uri         u:object_r:vendor_vis_prop:s0
+vendor.product.pvr.debug_level  u:object_r:graphics_config_prop:s0
```

Comment:
Could you please fix this incorrect end of the file?

Reply:
Done